packages/kbn-scout-reporting/src/reporting/index.ts

```diff
@@ -26,5 +26,7 @@ export const scoutPlaywrightReporter = (
 export const scoutFailedTestsReporter = (
   options?: ScoutPlaywrightReporterOptions
 ): ReporterDescription => {
-  return ['@kbn/scout-reporting/src/reporting/playwright/failed_test', options];
+  return SCOUT_REPORTER_ENABLED
+    ? ['@kbn/scout-reporting/src/reporting/playwright/failed_test', options]
```

Comment:
why is this an array?

Reply:
It is by design https://playwright.dev/docs/api/class-reporter

Reply:
Ok, I presume it's to contain more than one reporter much like other test runners.  Cool